nsv13/code/modules/overmap/fighters/modules/utility_modules.dm

```diff
@@ -0,0 +1,56 @@
+// Aircraft Plasma Cutter
+/obj/item/fighter_component/primary/plasmacutter
+	name = "217-A Heavy Plasma Cutter"
+	desc = "A modified plasma cutter for mounting on Sabre-class utility vessels."
+	icon_state = "mecha_plasmacutter"
+	item_state = "plasmacutter"
+	accepted_ammo = null
+	weight = 1 // Plasma cutter + gimbal mount, pretty heavy
+	burst_size = 1
+	fire_delay = 0.5 SECONDS
+	overmap_select_sound = 'nsv13/sound/effects/ship/pdc_start.ogg'
+	overmap_firing_sounds = list('sound/weapons/plasma_cutter.ogg')
+	icon = 'icons/mecha/mecha_equipment.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/guns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/guns_righthand.dmi'
+	slot = HARDPOINT_SLOT_UTILITY_PRIMARY
+	var/projectile = /obj/item/projectile/plasma/adv/sabre
+	var/charge_to_fire = 2500
+
+/obj/item/fighter_component/primary/plasmacutter/get_ammo()
+	var/obj/structure/overmap/small_craft/F = loc
+	if(!istype(F))
+		return FALSE
+	var/obj/item/fighter_component/battery/B = F.loadout.get_slot(HARDPOINT_SLOT_BATTERY)
```

Comment:
Using plasma as ammo would be more interesting imo, but we can use this for now.